tests/src/drivers/hash.c

```diff
@@ -13,8 +13,12 @@
 #include "test/drivers/hash.h"
 
 #if defined(MBEDTLS_TEST_LIBTESTDRIVER1)
+#if (MBEDTLS_VERSION_MAJOR == 3) && (MBEDTLS_VERSION_MINOR == 6)
```

Comment:
We will revisit that when we migrate the libtestdriver1 support to TF-PSA-Crypto and CMake. We may be able to get rid of the `#if (MBEDTLS_VERSION_MAJOR == 3) && (MBEDTLS_VERSION_MINOR == 6)` or what will replace it in this PR. Added a note `to https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/61`.